app/Testing/Context/InternalHttpRequesting.php

```diff
@@ -24,6 +24,22 @@ public function anUserIsRequesting(string $endpoint, string $method = Request::M
         $this->request = Request::create($endpoint, $method);
     }
 
+    /**
+     * @Given an user is requesting :url using :method
+     */
+    public function anUserIsRequestUsingMethod(string $endpoint, string $method): void
```

Comment:
You should not replicate method like that. Look at the method above, it's already prepared for handling http method. Just add `@Given an user is requesting :url using :method` in docblock for `anUserIsRequesting()` and it should works.

Reply:
Sure. I wasn't aware I can provide multiple docblock expressions back then.